data/en/isclosure.json

```diff
@@ -4,7 +4,7 @@
 	"syntax":"isClosure(object)",
 	"returns":"boolean",
 	"description":"Checks if a given object is a closure.",
-	"related": ["isCustomFunction"],
+	"related": ["isCustomFunction", "isClosure"],
```

Comment:
It doesn't make sense to have a function related to itself.

Reply:
Not only doesn't it make sense, test should fail anyway.

Reply:
Sorry, yes this is a stupid mistake on my part. I meant to link it to `isValid`.